src/mrack/providers/aws.py

```diff
@@ -370,6 +370,9 @@ async def create_server(self, req):
                 "MarketType": "spot",
             }
 
+        if specs.get("metadata"):
+            request["UserData"] = str(specs.get("metadata"))
```

Comment:
We should use tags for this feature, not UserData. UserDarta are data provided to e.g. cloud-init so that the vm itself can work with them - not our use case.